src/typeDefs/gqlTypes.ts

```diff
@@ -236,6 +236,7 @@ export enum Locale {
   ThTh = 'th_TH',
   TlPh = 'tl_PH',
   TrTr = 'tr_TR',
+  Und = 'und',
```

Comment:
nit: could add a comment to explain this is undefined since the shortened version not be intuitive to the reader.

Reply:
after talking to @NabbeunNabi it sounds like this is an official recommended value, so I would add that note to the comment

Reply:
my bad, this is a generated file and the comment is already on the schema file